ambry-router/src/main/java/com.github.ambry.router/DeleteManager.java

```diff
@@ -100,10 +100,11 @@ void submitDeleteBlobOperation(String blobIdString, FutureResult<Void> futureRes
     try {
       BlobId blobId = RouterUtils.getBlobIdFromString(blobIdString, clusterMap);
       DeleteOperation deleteOperation =
-          new DeleteOperation(routerConfig, responseHandler, blobId, futureResult, callback, time);
+          new DeleteOperation(routerConfig, responseHandler, blobId, futureResult, callback, time, routerMetrics);
```

Comment:
could you add `routerMetrics` as the second parameter, right after `routerConfig`? It makes sense to have the config and metrics together, and it is also consistent with pattern used elsewhere.